x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/data_quality_summary/index.tsx

```diff
@@ -6,108 +6,75 @@
  */
 
 import { EuiFlexGroup, EuiFlexItem, EuiPanel } from '@elastic/eui';
-import React, { useMemo } from 'react';
+import React from 'react';
 import styled from 'styled-components';
 
-import { getErrorSummaries } from '../../helpers';
 import { StatsRollup } from '../pattern/pattern_summary/stats_rollup';
 import { SummaryActions } from './summary_actions';
-import type { OnCheckCompleted, PatternRollup } from '../../types';
+import { IlmPhaseFilter } from './ilm_phase_filter';
+import { useDataQualityContext } from '../data_quality_context';
+import { useResultsRollupContext } from '../../contexts/results_rollup_context';
 
 const MAX_SUMMARY_ACTIONS_CONTAINER_WIDTH = 400;
 const MIN_SUMMARY_ACTIONS_CONTAINER_WIDTH = 235;
 
+const StyledFlexGroup = styled(EuiFlexGroup)`
+  min-height: calc(174px - ${({ theme }) => theme.eui.euiSizeL} * 2);
+`;
+
+const StyledFlexItem = styled(EuiFlexItem)`
+  gap: ${({ theme }) => theme.eui.euiSizeL};
+`;
+
 const SummaryActionsContainerFlexItem = styled(EuiFlexItem)`
   max-width: ${MAX_SUMMARY_ACTIONS_CONTAINER_WIDTH}px;
   min-width: ${MIN_SUMMARY_ACTIONS_CONTAINER_WIDTH}px;
-  padding-right: ${({ theme }) => theme.eui.euiSizeXL};
 `;
 
-export interface Props {
-  addSuccessToast: (toast: { title: string }) => void;
-  canUserCreateAndReadCases: () => boolean;
-  formatBytes: (value: number | undefined) => string;
-  formatNumber: (value: number | undefined) => string;
-  ilmPhases: string[];
-  lastChecked: string;
-  openCreateCaseFlyout: ({
-    comments,
-    headerContent,
-  }: {
-    comments: string[];
-    headerContent?: React.ReactNode;
-  }) => void;
-  patternIndexNames: Record<string, string[]>;
-  patternRollups: Record<string, PatternRollup>;
-  patterns: string[];
-  setLastChecked: (lastChecked: string) => void;
-  totalDocsCount: number | undefined;
-  totalIncompatible: number | undefined;
-  totalIndices: number | undefined;
-  totalIndicesChecked: number | undefined;
-  totalSizeInBytes: number | undefined;
-  onCheckCompleted: OnCheckCompleted;
-}
+const StyledIlmPhaseFilterContainer = styled.div`
+  width: 100%;
+  max-width: 432px;
```

Comment:
I would actually put those constants back into styles :D Because there is no reason for them to be outside besides enforcing the type, which to me is excessive here. Namewise it's clear what those variables are because the css rule explicitly states that those are `min-width` and `max-width`  and furthermore enclosing styledcomponent and name tells explicitly where they belong `SummaryActionsContainerFlexItem`

So to me this feels a little like:

```tsx
// this is a variable
const variable = 'variable'
```

:D

But good question 

Wdyt? 

cc: @angorayc